### ISSUE TITLE
bug: panic: open defsec\policies: file does not exist (Azure DevOps on Windows 2019)

### ISSUE THREAD

**Issue by vanmash** [aquasecurity/tfsec#1601]
**Describe the bug**

`tfsec` task fails in Azure DevOps pipelines

**To Reproduce**
1. run  `tfsec` scan in Azure DevOps on Windows 2019 agent (`windows-latest`)
2. See an error 
```
panic: open defsec\policies: file does not exist

goroutine 1 [running]:
github.com/aquasecurity/defsec/rego.init.0()
	/home/runner/go/pkg/mod/github.com/aquasecurity/defsec@v0.19.1/rego/embed.go:20 +0x414
##[error]Cmd.exe exited with code '2'.
```

**Expected behavior**

`tfsec` runs

**System Info**

 - tfsec version: `v1.9.1`
 - terraform version: `v1.1.7`
 - OS: `windows-2019` and `windows-2022`

**Additional context**

Azure DevOps task
```bash
tfsec terraform/ --format JUnit --out tfsec-report-$(Build.BuildId).xml
```

**Comment by liamg** [comment]
Fix incoming :+1:

**Comment by liamg** [comment]
TIL Windows doesn't like paths beginning with `./`. We have a working branch now, release should be available within 30 minutes.

**Comment by liamg** [comment]
Fixed in `v1.9.3`

**Comment by vanmash** [comment]
Like new. Thank you @liamg
